Fix sed Command Compatibility Issue in add-lang-to-versions-template-% Target on macOS

In the `add-lang-to-versions-template-%` Makefile target, the following error is produced on macOS.


```
sed: 1: "./_templates/versions.html": invalid command code .
make[2]: *** [add-lang-to-versions-template-af] Error 1
make[1]: *** [html] Error 2
make: *** [user-manual-html] Error 2
```

--- a/user_manual/Makefile
+++ b/user_manual/Makefile
@@ -68,8 +68,8 @@ merge-folders:
 	cd $(BUILDDIR)/html/ && rm -rf `find -type d -name _images`
 	mv ./_static $(BUILDDIR)/html/
 	mv ./_images $(BUILDDIR)/html/
-	cd $(BUILDDIR)/html/ && find ./ -type f -exec sed -i -e 's/_images/..\/_images/g' {} \;
-	cd $(BUILDDIR)/html/ && find ./ -type f -exec sed -i -e 's/_static/..\/_static/g' {} \;
+	cd $(BUILDDIR)/html/ && find ./ -type f -exec perl -pi -e 's|_images|../_images|g' {} \;
+	cd $(BUILDDIR)/html/ && find ./ -type f -exec perl -pi -e 's|_static|../_static|g' {} \;
 
 dirhtml:
 	$(SPHINXBUILD) -b dirhtml $(ALLSPHINXOPTS) $(BUILDDIR)/dirhtml

--- a/user_manual/Makefile
+++ b/user_manual/Makefile
@@ -69,8 +69,8 @@ merge-folders:
 	cd $(BUILDDIR)/html/ && rm -rf `find -type d -name _images`
 	mv ./_static $(BUILDDIR)/html/
 	mv ./_images $(BUILDDIR)/html/
-	cd $(BUILDDIR)/html/ && find ./ -type f -exec sed -i -e 's/_images/..\/_images/g' {} \;
-	cd $(BUILDDIR)/html/ && find ./ -type f -exec sed -i -e 's/_static/..\/_static/g' {} \;
+	cd $(BUILDDIR)/html/ && find ./ -type f -exec perl -pi -e 's|_images|../_images|g' {} \;
+	cd $(BUILDDIR)/html/ && find ./ -type f -exec perl -pi -e 's|_static|../_static|g' {} \;
 
 dirhtml:
 	$(SPHINXBUILD) -b dirhtml $(ALLSPHINXOPTS) $(BUILDDIR)/dirhtml

